grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -825,4 +839,26 @@ function AttachedIncidentsList({
   );
 }
 
+const AlertGroupStub = ({ buttons }: { buttons: React.ReactNode }) => {
```

Comment:
This looks like some reusable component that we might use for other cases too. It's might be worth to make it more flexible (it can accept texts as props) and put it in `components`

Reply:
let's extract it to reusable component once we need to reuse it